drivers/rtc/Kconfig

```diff
@@ -38,6 +38,7 @@ config RTC_CALIBRATION
 config RTC_SHELL
 	bool "RTC Shell commands"
 	depends on SHELL
+	imply DEVICE_DT_METADATA
```

Comment:
Not sure this should be implied, really.

Reply:
I simply copied this from GPIO_SHELL. Also,  to me it makes sense since shell is mainly for development so you would want the easiest access possible.

Reply:
I think its okay to imply this if the shell can make use of it :)